mythril/laser/ethereum/instructions.py

```diff
@@ -1842,6 +1842,14 @@ def call_(self, global_state: GlobalState) -> List[GlobalState]:
             callee_address, callee_account, call_data, value, gas, memory_out_offset, memory_out_size = get_call_parameters(
                 global_state, self.dynamic_loader, True
             )
+
+            if callee_account is not None and callee_account.code.bytecode == "":
```

Comment:
I don't think this edgecase should be implemented in the call instruction, I'd say that this belongs in the transaction handling